client/src/app/site/motions/modules/amendment-list/amendment-list.component.html

```diff
@@ -1,6 +1,14 @@
 <os-head-bar [nav]="false" [multiSelectMode]="isMultiSelect" goBack="true">
     <!-- Title -->
-    <div class="title-slot"><h2 translate>Amendments</h2></div>
+    <div class="title-slot" *ngIf="!parentMotion"><h2 translate>Amendments</h2></div>
+
+    <!-- TODO would require translations with parameters -->
+    <div class="title-slot" *ngIf="parentMotion">
+        <h2>
+            {{ 'Amendments to' | translate }}
+            {{ parentMotion | async }}
```

Comment:
Please show only the identifier of parent motion - without motion title!
I tried it with `parentMotion.identifier` - but it does not work...

Reply:
we access `parentMotion` asynchronously. Syntax would be: `{{ (parentMotion | async).identifier }}`